cmd/utils/flags.go

```diff
@@ -399,6 +399,10 @@ var (
 		Name:  "quaistats",
 		Usage: "Reporting URL of a quaistats service (nodename:secret@host:port)",
 	}
+	TrustedNodeFlag = cli.BoolFlag{
+		Name:  "trustednode",
+		Usage: "Truted node boolean flag for use by team",
```

Comment:
Please explain what 'trusted node' means. Everyone trusts their own node. Nobody should trust anyone elses node. What does this flag indicate?